CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,82 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```

Comment:
'ethnicity' instead of 'race' ?

Reply:
Ethnicity is listed on line 8.

Ethnicity and race are two different things - ie one based on heritage/culture and the other on physical characteristics. eg https://www.diffen.com/difference/Ethnicity_vs_Race

Reply:
:+1: